tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -259,6 +259,7 @@ public function usingOldDavPath():void {
 		$this->davPath = $this->getOldDavPath();
 		$this->usingOldDavPath = true;
 		$this->customDavPath = null;
+		$this->usingSpacesDavPath = false;
```

Comment:
do we need this?

Reply:
similar to other functions. I think it got missed out here.